src/main/resources/logback.xml

```diff
@@ -57,6 +57,11 @@
         <appender-ref ref="STDOUT"/>
     </logger>
 
+    <!-- TO SEE SQL QUERY LOG-OUTPUT SWITCH THIS LEVEL TO DEBUG-->
+    <logger name="scalikejdbc" level="INFO" additivity="false">
```

Comment:
not too much printing??

Reply:
No. Root was at INFO, too. So no change.
 If you change it (locally) to DEBUG, you get the SQL-statements.